packages/react-components/react-table/src/components/TableRow/useTableRowStyles.styles.ts

```diff
@@ -57,7 +58,7 @@ const useStyles = makeStyles({
   noAppearanceFocusWithin: {
     ...createCustomFocusIndicatorStyle(
       {
-        backgroundColor: tokens.colorSubtleBackgroundHover,
+        // backgroundColor: tokens.colorSubtleBackgroundHover,
```

Comment:
Cruft? Do we need to call `createCustomFocusIndicatorStyle` at all?

Reply:
Sorry, the PR was missing a commit - my bad 🙈